metadata-ingestion/src/datahub/ingestion/source/redshift/query.py

```diff
@@ -358,12 +360,12 @@ def list_unload_commands_sql(
             join stl_scan sc on
                 sc.query = unl.query and
                 sc.starttime >= '{start_time}' and
-                sc.endtime < '{end_time}'
+                sc.starttime < '{end_time}'
```

Comment:
nice, I thought we don't have this start_time endtime mismatch but it seems like there was still